pom.xml

```diff
@@ -11,7 +11,7 @@
   </parent>
 
   <artifactId>sonar</artifactId>
-  <version>${revision}${changelist}</version>
+  <version>2.18</version>
```

Comment:
I see that for the previous release one more place was updated: https://github.com/SonarSource/sonar-scanner-jenkins/commit/fc01fede701ba4d0b4181c375dd780a94f86487b